sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/README.md

```diff
@@ -3,9 +3,10 @@
 ## Key concepts
 
 This code sample demonstrates how to use the Spring Cloud Stream Binder
-for Azure Event Hub. The sample app exposes a RESTful API to receive string
-message. Then message is sent through Azure Event Hub to a `sink` which
-simply logs the message.
+for Azure Event Hub. The sample app has two operating modes, such as "manual" and "non-manual". 
+One way is to expose a Restful API to receive string message , another way is to automatically provide String messages.
+These Messages are added to the EventHub Stream. The sample application consumes the Messages from the stream.
```

Comment:
Same for two `Messages`.

Reply:
There's no such expression of `EventHub Stream`, we could change the sentence to 
```
These messages are published to an event hub. The sample will also consume messages from the same event hub.
```